x-pack/plugins/apm/public/components/app/transaction_details/WaterfallWithSummmary/TransactionTabs.tsx

```diff
@@ -137,6 +137,19 @@ function LogsTabContent({ transaction }: { transaction: Transaction }) {
       endTimestamp={endTimestamp + framePaddingMs}
       query={`trace.id:"${transaction.trace.id}" OR "${transaction.trace.id}"`}
       height={640}
+      columns={[
+        { type: 'timestamp' },
+        {
+          type: 'field',
+          field: 'service.name',
+          header: i18n.translate(
+            'xpack.apm.propertiesTable.tabs.logs.serviceName',
+            { defaultMessage: 'Service Name' }
+          ),
+          width: 200,
+        },
+        { type: 'message' },
```

Comment:
I'm wondering if we should also display `transaction.id` (i think it's more likely to be available than `service.name`) but this is a good start.